rpm_docker.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+sudo service docker start
+docker build -t centos:latest .
+docker cp `docker ps -l -q`:/root/rpmbuild/RPMS/x86_64/${PACKAGE}-${VERSION}-1.x86_64.rpm ./
```

Comment:
I have confirmed the error in my environment that @pastak has reported.
```bash
% docker build -t centos:latest .
...
It appears your locale is set to US-ASCII, please use UTF-8 instead. Run the 'locale' command to getmore info or e-mail support@packagecloud.io for help
INFO[0283] The command [/bin/sh -c package_cloud push imkitchen/gyazo/el/7 ./gyazo*.rpm] returned a non-zero code: 1 
```

In Dockerfile, you can use ``ENV`` statement as follow.
```dockerfile
# change locale to en_US.utf8
ENV LANG en_US.utf8
ENV LANGUAGE en_US:en
ENV LC_ALL en_US.utf8
```